### ISSUE TITLE
"search" argument of getDebates()

### ISSUE THREAD

**Issue by bellodi92** [jblumenau/twfyR#3]
When the argument “search” of getDebates() consists of multiple words that are not typed by hand (e.g., entered as object[i] of a vector) the call to the API does not replace \\s with +. Example below:
 
Error calling the following url: https://www.theyworkforyou.com/api/getDebates&output=xml&search=Agriculture and Horticulture Development Board&order=d&page=44&num=1000&type=commons&key=EeFvBfFd3b2CEznyiZE5eBt5

**Comment by jblumenau** [comment]
I realise this was some time ago, but I can't reproduce this error. 

````
> words <- c("Agriculture and Horticulture Development Board", "Something")
> test_speeches <- getDebates(type = "commons", search = words[1], order = "d")
> dim(test_speeches)
[1] 95 28
````

**Comment by bellodi92** [comment]
Right! This works just fine on my mac. If I run the same code on another PC, same R version, after having re-installed the package, I still get an error message:

`words <- c("Agriculture and Horticulture Development Board", "Something")`

 `test_speeches <- getDebates(type = "commons",
                                                    search = words[1],
                                                    order = "d")`

`Error in data_page_list[[doc]] : subscript out of bounds`

And if I add complete.call = T, I get the original error message I posted:

`test_speeches <- getDebates(type = "commons",
                            search = words[1],
                            order = "d",
                            complete_call = T)`

`Retreiving all the data for your search will require approximately 4 calls to the twfy API.
Call 1 complete.
Error calling the following url: https://www.theyworkforyou.com/api/getDebates&output=xml&search=Agricultureand Horticulture Development Board&order=d&page=2&num=1000&type=commons&key=EeFvBfFd3b2CEznyiZE5eBt5
Skipping to next call.`

If I replace \\s with +, it works fine. 

I'm pretty sure this is a problem of the PC though. 
Thanks
